src/functions/revealImg.tsx

```diff
@@ -0,0 +1,17 @@
+const revealImg = () => {
+  let reveals = document.querySelectorAll('.reveal-img');
```

Comment:
use const for all variables (except i) as there are no value changes done afterward